collect_app/src/main/java/org/odk/collect/android/dao/FormsDao.java

```diff
@@ -72,6 +72,17 @@ public Cursor getFormsCursor(String[] projection, String selection, String[] sel
     public void deleteFormsDatabase() {
         Collect.getInstance().getContentResolver().delete(FormsProviderAPI.FormsColumns.CONTENT_URI, null, null);
     }
+    
+    public void deleteFormsFromIDs(String[] idsToDelete) {
+
+        String selection = FormsProviderAPI.FormsColumns._ID + " in (";
+        for (int i = 0; i < idsToDelete.length - 1; i++)
+            selection += "?, ";
+        selection += "? )";
+        
+        Collect.getInstance().getContentResolver().delete(FormsProviderAPI.FormsColumns.CONTENT_URI, selection, idsToDelete);
```

Comment:
I know it's probably never gonna happen, but we probably need to do the slicing here too? The 999 issue thing that we have with the other sqlite operation with '?'